packages/webdriverio/src/commands/element/click.ts

```diff
@@ -125,10 +126,10 @@ export async function click(
     if (xOffset || yOffset) {
         const { width, height } = await browser.getElementRect(this.elementId)
         if ((xOffset && xOffset < (-Math.floor(width / 2))) || (xOffset && xOffset > Math.floor(width / 2))) {
-            throw new Error('xOffset would cause a out of bounds error as it goes outside of element')
+            log.info('xOffset would cause a out of bounds error as it goes outside of element')
```

Comment:
Let's make it a warn log here and in the other appearences.

```suggestion
            log.warn('xOffset would cause a out of bounds error as it goes outside of element')
```